### ISSUE TITLE
Caps lock does not modify number row

### ISSUE THREAD

**Issue by foxsouns** [jayliu50/windows-programmer-dvorak#2]
as shown [here](https://www.kaufmann.no/roland/dvorak/), the layout should have the number keys affected by the capslock key.

**Comment by jayliu50** [comment]
I agree there is a discrepancy. I don't think that what you are pointing out can be achieved with the toolset. However, I would be happy to be proven wrong.

**Comment by foxsouns** [comment]
I'm not familiar with Keyboard Layout Creator, but maybe [this](https://superuser.com/a/1333938) would apply?

**Comment by KhalidAlansary** [comment]
Did you figure it out?

**Comment by jayliu50** [comment]
I haven't had bandwidth to do so, but will happily review PRs

**Comment by jayliu50** [comment]
Thanks, @KhalidAlansary!
